### PR TITLE
183792092 merge leaders with map

### PR DESCRIPTION
**Pull request by kbiala**
#### What's this PR do?
- Moves leaders component to map

#### How should this be manually tested?
- On localhost run leaders daemon for selected network (eg. `rails r daemons/leader_stats_mainnet_update_daemon.rb`)
- Go to homepage and check the map

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket
](https://www.pivotaltracker.com/story/show/183792092)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: app/javascript/packs/validators/validators_map_leaders.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import Vue from 'vue/dist/vue.esm'
+
+var ValidatorsMapLeaders = Vue.component('ValidatorsMapLeaders', {
+    props: {
```
Comment (kamtarnowski) [inline app/javascript/packs/validators/validators_map_leaders.js:4]: Is there something wrong with the indentation?

File: app/views/public/home.html.erb [hunk 2]
```diff
@@ -15,11 +14,14 @@
   <validators-map network="<%= @network %>"/>
 </div>
 
+<!--
```
Comment (kamtarnowski) [inline app/views/public/home.html.erb:17]: Can we remove these unused lines of code?

**Review by kamtarnowski — CHANGES_REQUESTED** [review]
Please also remove 
`<%= javascript_pack_tag 'leaders/leaders_stats_component.js' %>`
from home_new.html.erb

**Comment by kamtarnowski** [comment]
The code looks pretty good, also the homepage seems much better.

**Comment by kbiala** [comment]
@kamtarnowski done

**Review by kamtarnowski — APPROVED** [review]

**Comment by kamtarnowski** [comment]
That is not a blocker, but maybe we could just remove this view - `app/views/public/home_new.html.erb`, to not maintain both similar views. Any reference to `home_new` path would be defined in the routes file.

**Comment by kbiala** [comment]
> That is not a blocker, but maybe we could just remove this view - `app/views/public/home_new.html.erb`, to not maintain both similar views. Any reference to `home_new` path would be defined in the routes file.

Very bad idea - this view was supposed to be different, to show things that are in progress and pending for acceptance.
Anyway, we probably don't need it at all anymore. But in any case it's completely unrelated to this story.

**Pull request merged**